hieradata/cluster/konkong.yaml

```diff
@@ -0,0 +1,131 @@
+---
+clustershell::groupmembers:
+  konkong: {group: "konkong", member: "konkong[01-03]"}
```

Comment:
Same as above, just suggesting to go 1-5

Reply:
It is better that the clustershell config only references nodes which exist. Otherwise, clush commands will always return an error for those nodes.